sql/core/src/test/scala/org/apache/spark/sql/execution/arrow/ArrowConvertersSuite.scala

```diff
@@ -21,24 +21,22 @@ import java.nio.charset.StandardCharsets
 import java.sql.{Date, Timestamp}
 import java.text.SimpleDateFormat
 import java.util.Locale
-
```

Comment:
Let's keep these newlines. I think Scala linter would complain about this.

Reply:
Thank you, I have reverted these changes.